apps/server/src/modules/learnroom/service/dashboard.service.ts

```diff
@@ -1,18 +1,24 @@
 import { Inject, Injectable } from '@nestjs/common';
 import { EntityId } from '@shared/domain/types';
 import { IDashboardRepo, DashboardElementRepo } from '@shared/repo';
+import { LegacyLogger } from '@src/core/logger';
 
 @Injectable()
 export class DashboardService {
 	constructor(
 		@Inject('DASHBOARD_REPO') private readonly dashboardRepo: IDashboardRepo,
-		private readonly dashboardElementRepo: DashboardElementRepo
-	) {}
+		private readonly dashboardElementRepo: DashboardElementRepo,
+		private readonly logger: LegacyLogger
+	) {
+		this.logger.setContext(DashboardService.name);
+	}
 
 	async deleteDashboardByUserId(userId: EntityId): Promise<number> {
+		this.logger.log({ action: 'Deleting dasboard for userId - ', userId });
 		const usersDashboard = await this.dashboardRepo.getUsersDashboard(userId);
 		await this.dashboardElementRepo.deleteByDashboardId(usersDashboard.id);
 		const result = await this.dashboardRepo.deleteDashboardByUserId(userId);
+		this.logger.log({ action: 'Deleted dasboard for userId - ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of deleted dashboard for userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114